doc/langref.html.in

```diff
@@ -1757,7 +1757,7 @@ const std = @import("std");
 const assert = std.debug.assert;
 
 test "anonymous list literal syntax" {
-    var array: [4]u8 = .{11, 22, 33, 44};
```

Comment:
The type here was intentional, note how the next paragraph says "If there is no type in the result location then..."

Reply:
Missed that, thanks. I'll drop this patch.